site/docs/latest/clients/WebSocket.md

```diff
@@ -132,6 +145,15 @@ The consumer endpoint requires you to specify a {% popover property %}, {% popov
 
 {% endpoint ws://broker-service-url:8080/ws/consumer/persistent/:property/:cluster/:namespace/:topic/:subscription %}
 
+##### Query param
+
+Key | Type | Required? | Explanation
+:---|:-----|:----------|:-----------
+`ackTimeoutMillis` | long | no | Set the timeout for unacked messages (default: 1000)
```

Comment:
It seems that the default value of `ackTimeoutMillis` is 0.
https://github.com/apache/incubator-pulsar/blob/bace0b47ada0d107f33d26ac282e1d6fb0a43156/pulsar-websocket/src/main/java/org/apache/pulsar/websocket/ConsumerHandler.java#L244
https://github.com/apache/incubator-pulsar/blob/bace0b47ada0d107f33d26ac282e1d6fb0a43156/pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerConfiguration.java#L51

Reply:
yes, by mistake I copied `long minAckTimeoutMillis = 1000` time. will fix it.